plans/remote-runner/README.md

```diff
@@ -0,0 +1,67 @@
+# Remote Runner Plan
+
+This is an example of a Testground plan to build an application with Nix and run
+the application on different machines. It's a prototype, so expect this to
+change and to be better supported in the future.
+
+At a high level, this works by:
+1. Using [Nix] to build the application for each machine's architecture
+   (e.g. One node may be `x86_64-linux` and another may be `aarch64-darwin`).
+   You may need to set up [remote
+   builders](https://nixos.org/manual/nix/stable/advanced-topics/distributed-builds.html)
+   to properly cross-build the application.
+1. Using a `local:exec` runner to run a shell script that copies the built
```

Comment:
```suggestion
2. Using a `local:exec` runner to run a shell script that copies the built
```
Numbering is wrong.

Also, would it be better define a new `remote:exec` runner?

Reply:
If you keep using `1.`s for list items then GitHub Markdown creates a numbered list starting from `1`. That's where the numbering comes from.

As for the name, `local:exec` is an existing construct. Also, the name is fitting here because the shell script that performs the copy is executed locally.